axum/src/routing/tests/mod.rs

```diff
@@ -720,3 +720,15 @@ async fn limited_body_with_streaming_body() {
         .await;
     assert_eq!(res.status(), StatusCode::PAYLOAD_TOO_LARGE);
 }
+
+#[tokio::test]
+async fn from_http_method() {
+    assert_eq!(MethodFilter::try_from(Method::DELETE).unwrap(), MethodFilter::DELETE);
+    assert_eq!(MethodFilter::try_from(Method::GET).unwrap(), MethodFilter::GET);
+    assert_eq!(MethodFilter::try_from(Method::HEAD).unwrap(), MethodFilter::HEAD);
+    assert_eq!(MethodFilter::try_from(Method::OPTIONS).unwrap(), MethodFilter::OPTIONS);
+    assert_eq!(MethodFilter::try_from(Method::PATCH).unwrap(), MethodFilter::PATCH);
+    assert_eq!(MethodFilter::try_from(Method::POST).unwrap(), MethodFilter::POST);
+    assert_eq!(MethodFilter::try_from(Method::PUT).unwrap(), MethodFilter::PUT);
+    assert_eq!(MethodFilter::try_from(Method::TRACE).unwrap(), MethodFilter::TRACE);
+}
```

Comment:
This doesn't need to be async.

```suggestion
#[test]
fn from_http_method() {
    assert_eq!(MethodFilter::try_from(Method::DELETE).unwrap(), MethodFilter::DELETE);
    assert_eq!(MethodFilter::try_from(Method::GET).unwrap(), MethodFilter::GET);
    assert_eq!(MethodFilter::try_from(Method::HEAD).unwrap(), MethodFilter::HEAD);
    assert_eq!(MethodFilter::try_from(Method::OPTIONS).unwrap(), MethodFilter::OPTIONS);
    assert_eq!(MethodFilter::try_from(Method::PATCH).unwrap(), MethodFilter::PATCH);
    assert_eq!(MethodFilter::try_from(Method::POST).unwrap(), MethodFilter::POST);
    assert_eq!(MethodFilter::try_from(Method::PUT).unwrap(), MethodFilter::PUT);
    assert_eq!(MethodFilter::try_from(Method::TRACE).unwrap(), MethodFilter::TRACE);
}
```

Reply:
You can also just click "Commit suggestion" to accept it, rather than typing it yourself.